motor.cpp

```diff
@@ -24,7 +24,7 @@ void Motor::stop(int number, int number2, int number3, int number4) {
 }
 
 void Motor::idz(int number, int dir, int speed) {
```

Comment:
I think we should rename these methods to English. Also, is there any better name for `number`? Because that one is pretty non-descriptive.